### DISCUSSION TITLE
Unable to import KingfisherSwiftUI despite using Kingfisher 6.1.1

### DISCUSSION THREAD

**Discussion by AustinConlon** [onevcat/Kingfisher#1777] [Q&A] [answered]
In a project that already uses Kingfisher 6.1.1, when trying to import KingfisherSwiftUI I get the error `No such module 'KingfisherSwiftUI'`. The [wiki](https://github.com/onevcat/Kingfisher/wiki/SwiftUI-Support) says that "No additional installation is required if you are using Kingfisher version 6." What might be the problem and potential fix?

**Comment by onevcat** [comment, marked as answer]
From version 6, the SwiftUI support is a part of the Kingfisher library, there is no standalone KingfisherSwiftUI anymore. So you just need `import Kingfisher` and use `KFImage`.

**Reply by AustinConlon** [reply]
> From version 6, the SwiftUI support is a part of the Kingfisher library, there is no standalone KingfisherSwiftUI anymore. So you just need `import Kingfisher` and use `KFImage`.

Thank you! This should be reflected in the wiki.

**Reply by onevcat** [reply]
If I am right, the description of `KingfisherSwiftUI` is under the supplementing section of "using SwiftUI in Kingfisher 5". Sorry for the confusion, I will consider to remove the outdated things soon.

**Comment by Sumeet-Mediacorp** [comment]
@onevcat  I'm facing below issue with Xcode 13, sorry for putting it here: 

*** Building scheme "KingfisherSwiftUI" in Kingfisher.xcworkspace
Build Failed
Task failed with exit code 65:
/usr/bin/xcrun xcodebuild -workspace /Users/admin/Documents/meWatch_iOS/Carthage/Checkouts/Kingfisher/Kingfisher.xcworkspace -scheme KingfisherSwiftUI -configuration Release -derivedDataPath /Users/admin/Library/Caches/org.carthage.CarthageKit/DerivedData/13.0_13A233/Kingfisher/5.15.6 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/ry/j50lnscs7p11r80mr5m82_8h0000gn/T/Kingfisher SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/admin/Documents/meWatch_iOS/Carthage/Checkouts/Kingfisher)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/ry/j50lnscs7p11r80mr5m82_8h0000gn/T/carthage-xcodebuild.sHIafd.log
admin@Admins-MacBook-Pro meWatch_iOS %
`
Did anyone else faced the same issue?

here is whats mentioned in the logs :

** ARCHIVE FAILED **

The following build commands failed:
CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler (in target 'KingfisherSwiftUI' from project 'Kingfisher')
CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler (in target 'KingfisherSwiftUI' from project 'Kingfisher')
CompileSwift normal armv7 (in target 'KingfisherSwiftUI' from project 'Kingfisher')
(3 failures)

**here is the original question:**  https://github.com/onevcat/Kingfisher/discussions/1813

**Reply by onevcat** [reply]
Duplicated to #1802
